addon/components/one-way-radio.js

```diff
@@ -25,28 +25,32 @@ const OneWayRadioComponent = Component.extend(DynamicAttributeBindings, {
     'type'
   ],
 
-  checked: computed('_value', 'option', function() {
-    return get(this, '_value') === get(this, 'option');
+  checked: computed('value', 'option', function() {
+    return this.get('value') === this.get('option');
   }),
 
-  click() {
-    invokeAction(this, 'update', get(this, 'option'));
+  click(e) {
+    if (!this.get('checked')) {
+      this._triggerChange();
+    }
+    return e.preventDefault();
   },
 
-  didReceiveAttrs() {
-    this._super(...arguments);
+  _triggerChange() {
+    invokeAction(this, 'update', get(this, 'option'));
+  },
 
-    let value = get(this, 'paramValue');
-    if (value === undefined) {
-      value = get(this, 'value');
+  didUpdateAttrs({ newAttrs }) {
```

Comment:
The params to lifecycle hooks got deprecated, so you'd need to refactor this to a way that don't use them.